<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaltPercentDialog</class>
 <widget class="QDialog" name="MaltPercentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QTableView" name="maltTable"/>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Target</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="weightCheck">
        <property name="text">
         <string>Weight</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="weightSpin"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="weightUnitsCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="ogCheck">
        <property name="text">
         <string>OG</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="ogSpin">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MaltPercentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MaltPercentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
